kbs/src/attestation/intel_trust_authority/mod.rs

```diff
@@ -287,8 +283,10 @@ impl Attest for IntelTrustAuthority {
 impl IntelTrustAuthority {
     pub async fn new(config: IntelTrustAuthorityConfig) -> Result<Self> {
         let token_verifier = JwkAttestationTokenVerifier::new(&AttestationTokenVerifierConfig {
-            attestation_token_type: AttestationTokenVerifierType::Jwk,
-            trusted_certs_paths: vec![config.certs_file.clone()],
+            extra_teekey_paths: vec![],
+            trusted_certs_paths: vec![],
+            trusted_jwk_sets: vec![config.certs_file.clone()],
+            insecure_key: true,
```

Comment:
```suggestion
            insecure_key: false,
```

Reply:
Let me fix this. Btw, do you think we need to rename `certs_file` to `jwkset` or something else?

Reply:
@mythi Ok. Let me response here. Actually for ITA the configure item here both are ok as it will not change the behavior. When I re-read the part of code, I have a question whether we need `token_verifier` object here inside ITA backend plugin code. I am afraid that we do not.

The logic here is 
- a user requests ITA for a token (and we do not need to care about the token trustiworthy as we just call the API of ITA)
- the user request KBS for resource (this time we should check the token trustiworthy)

wdyt?

Reply:
I had thought about the same but decided to keep it for now. We may have future feature additions like the faithful verification that further checks the token and it might be useful to do it here.